### ISSUE TITLE
Add docker port keyword to app service

### ISSUE THREAD

**Issue by isaacabraham** [CompositionalIT/farmer#869]
There's a secret squirrel keyword to map a docker port to the App Service port 80:

```fsharp
setting "WEBSITES_PORT" "8000" // maps port 8000 traffic from your docker container to port 80 for App Service
```

This took me about 5-10 minutes of googling. We should reduce this to 5-10 seconds by adding a new keyword to the webApp (and Functions) builders:

```fsharp
webApp {
    ....
    docker_port 8000
}
```

**Comment by stevebelskie** [comment]
@isaacabraham I'd be interested in submitting a PR for this. 

After a brief look at the code, I'm guessing the approach would be to add the `docker_port` operation to WebAppBuilder  (and a corresponding property in WebAppConfig) and handle adding the value to `this.CommonWebConfig.Settings` in the `IBuilder` implementation in  `WebAppConfig` (essentially how `docker_ci` works I think).  

And then testing the functionality - possibly in a test similar to this one: https://github.com/CompositionalIT/farmer/blob/3a882b7f694649b23499cd3e193ba91b4f9506b4/src/Tests/WebApp.fs#L408


Let me know if I'm on the right track or completely missing something!

**Comment by isaacabraham** [comment]
Yep, this feel pretty much it! There's already a couple of examples as you say - another one is [here](https://github.com/CompositionalIT/farmer/blob/master/src/Farmer/Builders/Builders.WebApp.fs#L270).

I'm not sure if this docker port thing also applies to Functions, or just Web Apps - if both, then instead of applying the code the web app config directly, you apply it to the [Common Web App Config](https://github.com/CompositionalIT/farmer/blob/master/src/Farmer/Builders/Builders.WebApp.fs#L165) which goes across both functions and web apps.
